tools/vim/Readme

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016 Marcin Copik
+#
+#  Distributed under the Boost Software License, Version 1.0. (See accompanying
+#  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+Automatically export configuration file for VIM plugin YouCompleteMe:
+https://github.com/Valloric/YouCompleteMe
+
+Installation:
+https://github.com/Valloric/YouCompleteMe
+
+Configuration:
+Configure hpx build, passing location for YCM configuration file with option
+HPX_WITH_VIM_YCM. File will be configured to use HPX compilation flags and
+placed in this directory. Configuration file should be placed in your working
+directory or in any directory above. An optimal choice may be the source
+directory. To avoid asking for confirmation and automatically load config file
```

Comment:
Since you suggest to place it in the source directory (which I am not in favor of), we should absolutely have a gitignore for that.